spec/models/ability_spec.rb

```diff
@@ -13,6 +13,18 @@
   let(:user) { nil }
   let(:another_user) { create(:external_user).user }
 
+  def self.agfs_claim_types
+    [Claim::AdvocateClaim, Claim::AdvocateInterimClaim, Claim::AdvocateSupplementaryClaim, Claim::AdvocateHardshipClaim]
```

Comment:
Could these not be loaded from your new shared context too?

Reply:
these helpers are not for use on examples but within a describe block for iteration over examples. But have pushed a commit to put these in shared context (cause it works and centralises the logic)